astropy/wcs/wcs.py

```diff
@@ -129,6 +130,8 @@
 # Additional relax bit flags
 WCSHDO_SIP = 0x10000
 
+# Regular expression defining SIP keyword
+SIP_KW = re.compile('''^(A|B|AP|BP)_(?:[0-9]|1[0-9])_(?:[0-9]|1[0-9])(|[A-Z])$''')
```

Comment:
@pllim @astrofrog @Xarthisius A possible comment would be 

"This regex matches the SIP coefficient names , `A_i_j<L>`, `B_i_j<L>`, `AP_i_j<L>`, `BP_i_j<L>`, where `i` and `j` are numbers and `L` is an optional letter referring to an alternate WCS) "